tests/test_advection.py

```diff
@@ -193,6 +195,44 @@ def test_advection_RK45(lon, lat, mode, rk45_tol):
     print(fieldset.RK45_tol)
 
 
+@pytest.mark.parametrize("mode", ["scipy", "jit"])
+def test_advection_3DCROCO(mode):
+    data_path = os.path.join(os.path.dirname(__file__), "test_data/")
+    fieldset = FieldSet.from_modulefile(data_path + "fieldset_CROCO3D.py")
```

Comment:
Use `from tests.utils import TEST_DATA` instead of creating `data_path`


```suggestion
    fieldset = FieldSet.from_modulefile(TEST_DATA / "fieldset_CROCO3D.py")
```